### PR TITLE
Adds CONCEPT_IN_SOURCE column

### PR DESCRIPTION
**Pull request by XachaB**
This adds a column for mapping from the glosses in the data to concepts, as in required in lexibank/johanssonsoundsymbolic/pull/1.

### CODE REVIEW

**Review by AnnikaTjuka — CHANGES_REQUESTED** [review]
Thanks, @XachaB !
Your updated file has quotation marks in it. Sometimes Excel adds them automatically ;) 
If you delete those the rest looks fine.

**Comment by XachaB** [comment]
removed

**Comment by XachaB** [comment]
hanks,

**Comment by XachaB** [comment]
closing was a mistake, I mis-clicked :)

File: concepticondata/conceptlists/Johansson-2020-344.tsv [hunk 1]
```diff
@@ -1,345 +1,346 @@
...
+Johansson-2020-344-273	273	YOUNG	1207	YOUNG	0	young
+Johansson-2020-344-274	274	1SG	1209	I	1	1SG
+Johansson-2020-344-275	275	1PLI	1131	WE (INCLUSIVE)	1	1PLI
+Johansson-2020-344-276	276	1PLE	1130	WE (EXCLUSIVE)	1	1PLE
+Johansson-2020-344-277	277	2SG	1215	THOU	1	2SG
+Johansson-2020-344-278	278	2PL	1213	YOU	1	2PL
+Johansson-2020-344-279	279	3SG	262	HE OR SHE OR IT	1	3SG
+Johansson-2020-344-280	280	3PL	817	THEY	0	3PL
+Johansson-2020-344-281	281	mother's mother (female speaking)			0	MM (fs)
+Johansson-2020-344-281	281	mother's mother (female speaking)			0	MM (fs)
```
Comment (AnnikaTjuka) [inline concepticondata/conceptlists/Johansson-2020-344.tsv:283]: That's strange, somehow there was a line added here.

Comment (XachaB) [reply concepticondata/conceptlists/Johansson-2020-344.tsv:283]: Thanks, I'll remove and check for other duplicates

**Comment by AnnikaTjuka** [comment]
One tip: you can check if there are any small mistakes (like the one above) with `concepticon test`.

**Comment by XachaB** [comment]
> One tip: you can check if there are any small mistakes (like the one above) with `concepticon test`.

I see, I imagine this comes from the "pyconcepticon" package ? I'll install it, check things and correct anything I find.

**Comment by AnnikaTjuka** [comment]
Yes, but everything looks good now. I'll wait until the Travis checks are completed and then approve.

**Comment by XachaB** [comment]
Thanks, I confirm that it does pass tests now. I'm glad you told me about it, if I need to do a PR later on, it'll be smoother from the start.

**Review by LinguList — APPROVED** [review]
Thanks this is what we need ;)

**Review by LinguList — COMMENTED** [review]
Ah, sorry, @chrzyki, sorry to bother you again here, but we discussed this issue in the context of lexibank, right? That is: the consistent naming. Could you remind us of what we agreed on? Was it "lexibank" as a simple column name?

**Comment by chrzyki** [comment]
Yes, we went for `LEXIBANK_GLOSS`: https://github.com/lexibank/lexibank/pull/238/files I'll also update the other concept lists based on this later on.

**Comment by XachaB** [comment]
Thanks, done. When this is merged, I can make a PR for the corresponding changes (it's small, but good practice) in the lexibank dataset.

File: concepticondata/conceptlists/Johansson-2020-344.tsv-metadata.json [hunk 2]
```diff
@@ -45,6 +45,10 @@
                     {
                         "datatype": "string", 
                         "name": "SOUND_SYMBOLIC"
+                    },
+                    {
+                        "datatype": "string", 
+                        "name": "CONCEPT_IN_SOURCE"
```
Comment (LinguList) [inline concepticondata/conceptlists/Johansson-2020-344.tsv-metadata.json:51]: of course, you need to change here as well!

**Review by LinguList — CHANGES_REQUESTED** [review]

**Comment by XachaB** [comment]
I'm not too sure how to tell github the changes have been done.

**Comment by XachaB** [comment]
Thanks

**Pull request merged**